src/components/TimeEstimator/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Time estimator.
+ * @copyright IBM Security 2019
+ */
+
+import Time16 from '@carbon/icons-react/lib/time/16';
+
+import classnames from 'classnames';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('time-estimator');
+
+/**
+ * Time estimators display the estimated amount of time it takes to complete a flow.
+ */
+const TimeEstimator = ({ children, className, ...props }) => (
+  <span className={classnames(namespace, className)} {...props}>
+    <Time16 className={`${namespace}__icon`} />
+    {children}
+  </span>
+);
+
+TimeEstimator.propTypes = {
+  /** Provide the contents of the `TimeEstimator` */
+  children: node.isRequired,
+
+  /** Provide an optional class to be applied to the containing node */
+  className: string,
```

Comment:
Don't we usually have a defaults for non-required props? (Usually the linter complains right?)

Reply:
I was hoping you'd catch this so we could discuss it!

For this one, I replicated how Carbon was handling their optional class names, but it turns out they don't provide anything - github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Accordion/Accordion.js#L27

I spoke with Josh to understand if this was an explicit decision, but I don't think it was.

We have used an empty string in the past (`''`), but I'm not sure if that's correct, as it should really be a `null` value to truly define it as a non-value.

What are your thoughts here? I'm leaning towards having a default but setting it to `null`.

Reply:
Oh thank you -- yeah that makes sense. 🤔 I would prefer `null` here instead of not having a default, if that's all right with you.